### ISSUE TITLE
Change alt to aria-label for SVG Block-I

### ISSUE THREAD

**Issue by bryanjonker-illinois** [ATLAS-Illinois/Illinois-Framework-Theme#1360]
On the Student Affairs site, “Student Engagement”, https://studentengagement.illinois.edu/, the I-Block at the top of the page is a SVG image. As shown in this screenshot, the SVG is using ALT text, however, ALT doesn’t work with SVG. Instead, aria-label should be used to provide an accessible name for the SVG.

<img width="1150" height="108" alt="Image" src="https://github.com/user-attachments/assets/6385f5db-8b46-48cb-941f-53a7a72d4e8c" />

**Comment by bryanjonker-illinois** [comment]
It looks like there's a modified toolkit that is causing this. Sending to Drupal team? 

https://studentengage23.web.illinois.edu/themes/custom/sasubtheme/js/modified-toolkit-3.0.js
